--- udftools-cvs20030825/Makefile.am.orig	2003-08-26 13:27:49.000000000 +0200
+++ udftools-cvs20030825/Makefile.am	2003-08-26 14:00:10.000000000 +0200
@@ -1 +1 @@
-SUBDIRS = libudffs mkudffs cdrwtool pktsetup udffsck wrudf doc udfct
+SUBDIRS = libudffs mkudffs cdrwtool pktsetup doc udfct
--- udftools-1.0.0b3/configure.in.orig	2003-10-24 03:52:14.000000000 +0200
+++ udftools-1.0.0b3/configure.in	2004-03-25 13:13:27.021388016 +0100
@@ -24,4 +24,4 @@
 AC_CHECK_FUNCS(gettimeofday mktime open64 lseek64 llseek)
 AC_CHECK_FUNCS([getopt_long],[],[AC_LIBOBJ([getopt])])
 
-AC_OUTPUT(Makefile libudffs/Makefile mkudffs/Makefile cdrwtool/Makefile pktsetup/Makefile udffsck/Makefile wrudf/Makefile doc/Makefile udfct/Makefile)
+AC_OUTPUT(Makefile libudffs/Makefile mkudffs/Makefile cdrwtool/Makefile pktsetup/Makefile doc/Makefile udfct/Makefile)
